apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -11,47 +11,113 @@
         <v-list-item-avatar>
-          <v-icon small>mdi-television-guide</v-icon>
+          <v-icon small>mdi-triangle-outline</v-icon>
         </v-list-item-avatar>
-
         <v-list-item-content>
-          <v-list-item-title>Results</v-list-item-title>
+          <v-list-item-title>
+            Results Comparison View
+          </v-list-item-title>
         </v-list-item-content>
       </v-list-item>
-      <v-list-item to="/compare" title="Compare evaluations' controls">
+      <v-list-item to="/profiles" title="View controls' results">
         <v-list-item-avatar>
-          <v-icon small>mdi-triangle-outline</v-icon>
+          <v-icon small>mdi-television-guide</v-icon>
         </v-list-item-avatar>
-
         <v-list-item-content>
-          <v-list-item-title>Comparison</v-list-item-title>
+          <v-list-item-title>
+            Profile View
+          </v-list-item-title>
         </v-list-item-content>
       </v-list-item>
+      <v-subheader>Files</v-subheader>
+      <v-expansion-panels flat="true" v-model="file_views">
+        <v-expansion-panel>
+          <v-expansion-panel-header title="View controls' results">
+            <v-list-item>Results</v-list-item>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content>
+            <div
+              v-if="
+                visible_evaluation_files != null &&
+                  visible_evaluation_files.length > 0
+              "
+            >
+              <FileItem
+                v-for="(file, i) in visible_evaluation_files"
+                :key="i"
+                :file="file"
+              />
+              <v-list-item
+                @click="toggle_all_evaluations"
+                title="Show all files' controls"
+              >
+                <v-list-item-avatar>
+                  <v-icon small>mdi-format-list-bulleted</v-icon>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    <div v-if="all_toggled_evaluation">
+                      Deselect all results set
+                    </div>
+                    <div v-else>Select all results set</div>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </div>
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+        <v-expansion-panel>
+          <v-expansion-panel-header title="View controls' results">
+            <v-list-item>Profiles</v-list-item>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content>
+            <div
+              v-if="
+                visible_profile_files != null &&
+                  visible_profile_files.length > 0
+              "
+            >
+              <ProfileItem
+                v-for="(file, i) in visible_profile_files"
+                :key="i"
+                :file="file"
+              />
```

Comment:
This seems extremely similar to FileItem, can it just be reused?

Reply:
These four lines are similar, yes.  Also, the code in SidebarFile and SidebarProfile (which is referenced by FileItem and ProfileItem) are currently similar.  This separation will set up addressing issue #82, where the profile view and results view will differ.

Reply:
The code in SidebarFile and SidebarProfile should not really be different even after the Results view is broken out into Results and Profiles, the only difference should be the path (which can just be an if statement). This should not be duplicated into 2 components.